app/__init__.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from flask_login import LoginManager
```

Comment:
These are new packages - good to add these to the `requirements.txt`. I checked out this commit to test, and needed to add these packages.

You can just copy these to the bottom of `requirements.txt` so we can `pip install` more easily when developing/deploying the app.

```
flask_sqlalchemy==3.1.1
flask_migrate==4.0.7
flask_login==0.6.3
```